scripts/configure_motor.py

```diff
@@ -1,18 +1,69 @@
 from pypot.feetech import FeetechSTS3215IO
 import argparse
 import time
+import serial.tools.list_ports
 
 DEFAULT_ID = 1  # A brand new motor should have id 1
 
+# These are placeholders. Please replace these with your actual device's USB IDs.
```

Comment:
So these are the IDs of the BOM's waveshare boards ?

Reply:
those are the vendor ID and product ID of the waveshare boards, if you do an `lsusb` you will see those printed out:
```bash
Bus 004 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 003 Device 010: ID 1a86:55d3 QinHeng Electronics USB Single Serial
Bus 003 Device 004: ID 8087:0026 Intel Corp. AX201 Bluetooth
Bus 003 Device 003: ID 0c45:672e Microdia Integrated_Webcam_HD
Bus 003 Device 002: ID 27c6:63ac Shenzhen Goodix Technology Co.,Ltd. Goodix USB2.0 MISC
Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
```
Like this... the QinHeng Electronics USB Single Serial is how mine shows up.

Reply:
So if you try it with yours and the Vid:Pid comes out different it can be added to that table and it will be detected.

Reply:
Ok, I'll try that on a single motor next week before merging !